### PR TITLE
Split witness calculation step for circom 2

### PR DESCRIPTION
**Pull request by nalinbhardwaj**
Fixes witness calculation for circom 2 wasms. Fixes https://github.com/iden3/snarkjs/issues/107, requires https://github.com/iden3/circom_runtime/pull/14 to be merged first and upgraded here

cc @phated

### CODE REVIEW

**Comment by phated** [comment]
Requires #118

**Comment by nalinbhardwaj** [comment]
Rebased on new master and tested with both Circom 1 and Circom 2 generated WASMs ✅. Ready for merge.

File: src/wtns_calculate.js [hunk 1]
```diff
@@ -29,11 +29,19 @@ export default async function wtnsCalculate(input, wasmFileName, wtnsFileName, o
...
 
-    const fdWtns = await binFileUtils.createBinFile(wtnsFileName, "wtns", 2, 2);
+        const fdWtns = await binFileUtils.createBinFile(wtnsFileName, "wtns", 2, 2);
 
-    await wtnsUtils.writeBin(fdWtns, w, wc.prime);
-    await fdWtns.close();
+        await wtnsUtils.writeBin(fdWtns, w, wc.prime);
+        await fdWtns.close();
+    } else {
+        const fdWtns = await fastFile.createOverride(wtnsFileName);
```
Comment (phated) [inline src/wtns_calculate.js:40]: @nalinbhardwaj Where is `fdWtns` actually being used?

Comment (phated) [reply src/wtns_calculate.js:40]: Oh, I see, it is having the wtnsbin inserted into it.

Comment (nalinbhardwaj) [reply src/wtns_calculate.js:40]: Yep, instead of going through the binFileUtils helpers, circom 2 witnesses write straight to the witness file.

**Pull request merged**